src/Arcus.Observability.Telemetry.AzureFunctions/Extensions/ILoggerBuilderExtensions.cs

```diff
@@ -37,5 +38,32 @@ public static ILoggingBuilder ClearProvidersExceptFunctionProviders(this ILoggin
 
             return loggingBuilder;
         }
+
+        /// <summary>
+        /// Removes Microsoft's 'ApplicationInsightsLoggerProvider' type from the registered logging providers
+        /// so that Arcus' Serilog Application Insights sink is the only one place telemetry is created and sent to Microsoft Application Insights.
+        /// </summary>
+        /// <param name="builder">The builder instance to add and remove logging providers.</param>
+        /// <returns>A <see cref="ILoggingBuilder"/> without Microsoft's 'ApplicationInsightsLoggerProvider' registration.</returns>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="builder"/> is <c>null</c>.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when no 'ApplicationInsightsLoggerProvider' instance can be found in the <paramref name="builder"/>.</exception>
+        public static ILoggingBuilder RemoveMicrosoftApplicationInsightsLoggerProvider(this ILoggingBuilder builder)
+        {
+            Guard.NotNull(builder, nameof(builder), "Requires an instance of the logging builder to remove Microsoft's 'ApplicationInsightsLoggerProvider'");
+
+            ServiceDescriptor descriptor = 
+                builder.Services.FirstOrDefault(service => service.ImplementationType?.Name == "ApplicationInsightsLoggerProvider");
+            
+            if (descriptor is null)
+            {
+                throw new InvalidOperationException(
```

Comment:
I wonder why we throw an exception if we cannot find something that we want to remove .... 

The only thing I can think of, is to make sure that the code fails should MSFT decide to rename this servcie, which makes it valid to throw then ....

Reply:
Yes, it's for the reason you mentioned. If we don't do anything in this case, the failure may be 'swallowed' and we would probably have obscure failures (like faulty operation names mentioned in #364 ).

Reply:
On the other hand, you could argue that this is something that we should detect with a test.

How is this Application Insights provider being added to the DI container ?  Is it automatically there ?  Is it added via a call to the `AddApplicationInsights` method ?  What if we have a situation where the App Insights provider is actually not there ?  Do we need to fail on that ?

Reply:
Yeah, it's by default there, even without any app settings or startup content. If we start investigating in out-of-process Azure Functions, we can come back on this. Maybe in those cases, it would not be needed.

Reply:
I'm just reluctant to throw here if this situation arises.
It means that your complete Function won't be able to work, just because your logging is maybe not working optimally.  I think it might be better to log a Warning instead.

Reply:
Yeah, I don't think we can log here already bc the application is not yet build up. Maybe we can log a DEBUG trace, though.

Reply:
The alternative, is that the logging is corrupt if we don't throw.

Reply:
A Debug Trace will not be logged when your loglevel is set to Information or Warning , which is most likely the case for production API's.   Therefore, I'd log a warning or error.

The alternative is corrupt logging, but a failing API and a support call for such an issue is also not something you want.  On the other hand, these are things that should pop-up during development / testing.

Reply:
Yeah, but like I said: we cannot log an error/warning here the 'normal' way because the logging is at this place still being constructed. We should look for other ways to inform the consumer. A DEBUG trace is an option, not sure what's left.

Reply:
Had a chat with @gverstraete .  Conclusion was that, if the provider is not there, the API will fail to start.  Since it is at startup time, it should be obvious that something is wrong.  Chances that this occurs on a restart of a working API are rather scarce.   
Hence, we'll just throw.

Reply:
Ok, great! Thx also @gverstraete to look into this.